ext/civiimport/Civi/BAO/Import.php

```diff
@@ -33,7 +33,7 @@ class Import extends CRM_Core_DAO {
    *
    * @var int
    */
-  protected $_id;
+  public $_id;
```

Comment:
why does this become public?

Reply:
The primary key must be accessible in order for all BAO actions to work. See `SearchBatchTest::testMetadata`